internal/service/storage/storage_backup.go

```diff
@@ -75,6 +75,40 @@ func (r *storageBackupResource) Schema(_ context.Context, _ resource.SchemaReque
 					stringplanmodifier.UseStateForUnknown(),
 				},
 			},
+			"encrypt": schema.BoolAttribute{
+				MarkdownDescription: encryptDescription,
+				Computed:            true,
+			},
+			"labels":        utils.ReadOnlyLabelsAttribute("storage"),
+			"system_labels": utils.SystemLabelsAttribute("storage"),
+			"size": schema.Int64Attribute{
+				MarkdownDescription: sizeDescription,
+				Computed:            true,
+				PlanModifiers: []planmodifier.Int64{
+					int64planmodifier.UseStateForUnknown(),
+				},
+			},
+			"tier": schema.StringAttribute{
+				MarkdownDescription: tierDescription,
+				Computed:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"type": schema.StringAttribute{
+				MarkdownDescription: typeDescription,
+				Computed:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"zone": schema.StringAttribute{
+				MarkdownDescription: "The zone the storage is in, e.g. `de-fra1`. You can list available zones with `upctl zone list`.",
```

Comment:
Could omit the list command from here (similarly as with templates) as zone is computed only 🤔 

```suggestion
				MarkdownDescription: "The zone the storage is in, e.g. `de-fra1`.",
```